ReactWindows/ReactNative/Touch/TouchHandler.cs

```diff
@@ -323,6 +362,29 @@ private static bool ShouldSendEnterLeaveEvent(DependencyObject view)
             return true;
         }
 
+        private static bool ShouldSendPointerMoveEvent(DependencyObject view)
```

Comment:
>ShouldSendPointerMoveEvent [](start = 28, length = 26)

Does this take into account any of the details of `pointerEvents`?

Reply:
E.g., `pointerEvents == none` or `pointerEvents == box-only`?

---
In reply to: [186078877](https://github.com/Microsoft/react-native-windows/pull/1787#discussion_r186078877) [](ancestors = 186078877)